resources/serverless/charts/docker-registry/templates/job.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if eq .Values.storage "filesystem" }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ template "docker-registry.name" . }}-image-gc
+spec:
+  schedule: "42 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          annotations:
+            sidecar.istio.io/inject: "false"
+        spec:
+          serviceAccountName: {{ template "docker-registry.name" . }}
+          restartPolicy: Never
+          containers:
+          - name: function-image-gc
```

Comment:
I have problem with this name. It doesn't clearly says what this job do. 
I though about name like `function-image-untagger`.